src/central-server/admin-service/ui/src/views/Settings/SystemSettings/ManagementServices.vue

```diff
@@ -0,0 +1,212 @@
+
+          <tr>
+            <td>
+              {{ $t('systemSettings.serviceProviderName') }}
+            </td>
+            <td
+              data-test="system-settings-management-service-provider-name-field"
+            >
+              {{ managementServicesConfiguration.service_provider_name }}
+            </td>
+            <td></td>
+          </tr>
+
+          <tr>
+            <td>
+              {{ $t('systemSettings.managementServiceSecurityServer') }}
+            </td>
+            <td data-test="system-settings-management-security-server-field">
+              {{ managementServicesConfiguration.security_server_id }}
+            </td>
+            <td></td>
+          </tr>
+
+          <tr>
+            <td>
+              {{ $t('systemSettings.wsdlAddress') }}
+            </td>
+            <td data-test="system-settings-management-wsdl-address-field">
+              {{ managementServicesConfiguration.wsdl_address }}
+            </td>
+            <td class="action-cell">
+              <xrd-button
+                v-if="managementServicesConfiguration.wsdl_address"
+                text
+                :outlined="false"
+                class="copy-button"
+                @click.prevent="
+                  copyUrl(managementServicesConfiguration.wsdl_address)
+                "
+                ><v-icon class="xrd-large-button-icon">icon-Copy</v-icon
+                >{{ $t('action.copy') }}</xrd-button
+              >
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              {{ $t('systemSettings.centralServerAddress') }}
+            </td>
+            <td>
+              {{ managementServicesConfiguration.services_address }}
+            </td>
+            <td class="action-cell">
+              <xrd-button
+                v-if="managementServicesConfiguration.services_address"
+                text
+                :outlined="false"
+                class="copy-button"
+                @click.prevent="
+                  copyUrl(managementServicesConfiguration.services_address)
+                "
+                ><v-icon class="xrd-large-button-icon">icon-Copy</v-icon
+                >{{ $t('action.copy') }}</xrd-button
+              >
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              {{ $t('systemSettings.securityServerOwnerGroupCode') }}
+            </td>
+            <td>
+              {{
+                managementServicesConfiguration.security_server_owners_global_group_code
+              }}
+            </td>
+            <td></td>
+          </tr>
+        </tbody>
+      </table>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { ManagementServicesConfiguration } from '@/openapi-types';
+import { toClipboard } from '@/util/helpers';
+import Vue from 'vue';
+import { mapStores } from 'pinia';
+import { managementServicesStore } from '@/store/modules/management-services';
+
+export default Vue.extend({
```

Comment:
ManagementServices.vue is component so it should go to -> src/components/systemSettings
Leaving only SystemSettings.vue in src/views/Settings/SystemsSettings as this the only one which is view